src/pages/Donate/DonateForm.vue

```diff
@@ -66,11 +71,30 @@ export default {
 		formDisclaimer: {
 			type: String,
 			default: '',
-		}
+		},
+		formSubmitAnalytics: {
+			type: Object,
+			default: () => {
+				return {
+					category: 'Donate Form',
+					action: 'click-donate-support-us-form',
+					label: 'Donate',
```

Comment:
I think you can remove the label property from here and the other components, since the analytics should always use buttonText for the analytics event

Reply:
yea, I agree and waffled on that a bit... will add a default for the buttonText.